surface/include/pcl/surface/impl/reconstruction.hpp

```diff
@@ -78,7 +78,7 @@ SurfaceReconstruction<PointInT>::reconstruct (pcl::PolygonMesh &output)
   }
 
   // Set up the output dataset
-  pcl::toPCLPointCloud2 (*input_, output.cloud); /// NOTE: passing in boost shared pointer with * as const& should be OK here
```

Comment:
It happened in a `clang-tidy` pass, I'm not entirely sure why

Reply:
If you don't know why it was removed, please restore it (in all files). I also have no idea why `clang-tidy` wouldn't like it

Reply:
Probably because it's within a `namespace pcl` and therefore redundant; although why it would leave other nearby constructs qualified with `pcl::` is mystifying